### PR TITLE
Cookiecutter: initialize folder setup

### PR DESCRIPTION
**Pull request by bobleesj**
@sbillinge  Please let me know if there are too much changes per a single PR. 

In the following PR, I will focus on CI/Pytest. I recall 3 out of 4 tests failed. I will have to check this thoroughly.

### CODE REVIEW

File: doc/Makefile
Comment (bobleesj) [inline doc/Makefile:1]: I had to change the folder name from `docs` to `doc`, adding 100k of lines being added. wonder if there is a workaround for this.

File: LICENSE.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
+
+Copyright (c) 2024, The Trustees of Columbia University
```
Comment (sbillinge) [inline LICENSE.rst:3]: I think this needs to go back further to when this was first worked on.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+.. |PythonVersion| image:: https://img.shields.io/pypi/pyversions/diffpy.nmf_mapping
+        :target: https://pypi.org/project/diffpy.nmf_mapping/
+
+.. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
+        :target: https://github.com/diffpy/diffpy.nmf_mapping/issues
+
+Run NMF analysis on PDF and XRD data
+
+* LONGER DESCRIPTION HERE
```
Comment (sbillinge) [inline README.rst:40]: This doesn't seem to have been taken care of.

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+For more information about the diffpy.nmf_mapping library, please consult our `online documentation <https://diffpy.github.io/diffpy.nmf_mapping>`_.
+
+Citation
+--------
+
+If you use diffpy.nmf_mapping in a scientific publication, we would like you to cite this package as
+
+        diffpy.nmf_mapping Package, https://github.com/diffpy/diffpy.nmf_mapping
```
Comment (sbillinge) [inline README.rst:49]: I think this has a paper that needs to be cited.

**Review by sbillinge — COMMENTED** [review]
yes, this is too much on one PR.  Please break it out.  Each part doesn't have to pass tests.

Do the big moves (docs -> doc for example) on a separate PR.  I am used to seeing this show up as a file rename rather than as a new set of files appearing in the document.  Make sure you use `git mv` and don't copy files.

**Comment by sbillinge** [comment]
@bobleesj closing this.  Please see comment.

**Comment by bobleesj** [comment]
Noted. I will try using `git mv` make smaller PRs.

File: README.rst (hunk 2 above)
Comment (bobleesj) [reply README.rst:40]: I will address this in a separate PR.

**Pull request closed without merging**